test/dev/dev.test.ts

```diff
@@ -0,0 +1,68 @@
+const path = require('path');
+
+const execa = require('execa');
+const {readdirSync, readFileSync, statSync, existsSync} = require('fs');
+const glob = require('glob');
+const os = require('os');
+const got = require('got');
+
+describe('snowpack dev', () => {
+  // tests don't run on windows. `snowpackProcess` does not get ended correctly
+  if (process.platform === 'win32') {
+    it.todo('snowpack dev tests are currently not running on Windows');
+    return;
+  }
+
+  let snowpackProcess;
+  afterEach(async () => {
+    snowpackProcess.cancel();
+    snowpackProcess.kill('SIGTERM', {
+      forceKillAfterTimeout: 2000,
+    });
+
+    try {
+      await snowpackProcess;
```

Comment:
can we just set a timeout of 2 seconds here, and then assume it's dead? Jest completing should close Node.js, which should close this process as well.

Thinking that your "failure to kill" issue may just be that this try-catch behaves differently on Node & Linux

Reply:
I fear that just assuming the process is terminated will result in the same problem where `jest` complaints that processes have not been ended correctly

I got burned by `shell: true` in the past, but happy to give it a try, whatever works at this point :D I'll try `windowsHide` too